src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -79,10 +81,23 @@ public void addMessage(Message message) {
   public List<Message> getMessagesInConversation(UUID conversationId) {
 
     List<Message> messagesInConversation = new ArrayList<>();
+    HashSet<UUID>idSet = new HashSet<UUID>();
```

Comment:
For naming sets, lists, etc. were generally simply do <name>s, so in this case, messageIds